Final_Submission

nbatista0630 opened the pull request:
5/7/2015

nbatista0630 committed:
Final-Submission

Pressure Effects of Dimple-Like Ridges
Batista, Nicholas
5/7/2015

nbatista0630 committed:
Final_Submission_2

Included presentation slides

labarba commented:
We can't see Chart 1 nor the Figures on the nbviewer-rendered notebook. I am guessing you worked on a Windows machine, right?

For each figure, I searched your source code for the figure caption to find the embed code, as follows:

Figure 1: NACA0010-35 Symmetric Airfoil
`<img src=\"./resources_foils/pictures/naca0010_35.png\">`

Figure 2: Modified Airfoil for Baseline Analysis
`<img src=\"./resources_foils/pictures/sym_build.png\">`

Figure 3: Modified Foil & Spherical Dimples
`<img src=\"./resources_foils/gifs/foils.gif\">`

Chart 1: Dimple Size Chart
`<img src=\"./resources_foils/pictures/dimple_size_chart.jpg\">`

Figure 4: Near-Wall Vortices Visualization
`<img src=\"./resources_foils/pictures/beratlis.png\">`

Figure 5: XFLR5 Pressure/Force Visualization
`<img src=\"./resources_foils/gifs/xflr5.gif\">`

Figure 6: Coefficient of Pressure vs. Panel Location
`<img src=\"./resources_foils/gifs/streamlines.gif\">`

Figure 7: Streamlines around Modified Airfoil
`<img src=\"./resources_foils/gifs/streamlines.gif\">`

Figure 8: Pressure Contour Plot
`<img src=\"./resources_foils/gifs/pressure_contour.gif\">`

Then in your repo, we see these file names in folder /resources_foils/pictures

```
naca0010_35.PNG
sym_build.PNG
dimple_size_chart.JPG
beratlis.PNG
```


Notice the difference? The extension of these files is upper case. Well, Windows is not case-sensitive, so it is able to find the files and display them (so is Mac OSX), but Linux is case-sensitive, and since nbviewer is on a Linux server, it is not finding your embedded images!

I'm not sure what's the issue with the .gif files, though. I can see them in the GitHub rendered notebook, but not on the nbviewer.

labarba commented:
Please edit the name of the files so that the extension is lower case, and commit that change, if you can.

nbatista0630 committed:
5-18-2015_QuickChange

Updated file names. Embedded video files as a possible alternative to
.gif files that are un-viewable in nbviewer.

nbatista0630 commented:
Hello Dr. Barba,
I committed that change earlier today. Please let me know if the changes
did not appear on the github repo.
Thank you,
Nick Batista

On Mon, May 18, 2015 at 2:08 PM, Lorena A. Barba <notifications@github.com>
wrote:

> Please edit the name of the files so that the extension is lower case, and
> commit that change, if you can.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/barbagroup/AeroPython-studentprojects/pull/8#issuecomment-103151799>
> .
>

labarba commented:
For some reason, it didn't work. Oh well. (Also, the GIF animations worked fine before!)

Applications for dimples include golf balls, helmets, bicycle helmets and automotive spoilers —> It would've been interesting to know some details of the flow regime in these cases (Reynolds number and perhaps the relative size of the dimples). Otherwise, it's not clear how the situation studied in this project relates to those applications.

It looks like you are using the word "visualization" when you really mean "simulation."

"These results indicate that dimpled roughness elements can induce pressure forces on an aerodynamic body." —>This is an obvious and superfluous statement!

I really miss a conclusion that tells me what you learned from this exercise. What insights can be gained from this analysis? This report certainly does not give a clear idea of that.

You have this bit of code in cell [58]:

```
accuracy=sum([panel.source_strength*panel.length for panel in panels])
acc=accuracy*100
```

How do you figure that multiplying the sum of panel strengths by 100 will give you a percentage? A percentage of what?

ncclementi commented:
I was curious about the images and gifs issue, so I tried to open the notebook. I'm able to see all images and gif in github and NBviewer. I tried with Firefox and Chrome, and no problems. Besides, I'm using a linux machine.
